tsconfig.json

```diff
@@ -5,24 +5,19 @@
     "target": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
     "jsx": "react-jsx",
-    "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "isolatedModules": true,
     "resolveJsonModule": true,
     "noEmit": true,
     "incremental": true,
     "sourceMap": true,
-    "declaration": true,
-    "declarationMap": true,
     "removeComments": true,
     "strict": true,
     "noImplicitOverride": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "forceConsistentCasingInFileNames": true,
-    "importsNotUsedAsValues": "error",
-    "ignoreDeprecations": "5.0",
```

Comment:
I linked to an article that discussed this, I think: https://johnnyreilly.com/typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports -- it's ESLint's job